testing_dir/minesweeper.py

```diff
@@ -36,7 +36,7 @@ def print_stats():
 def update_player_pos(direction):
     x, y = player_pos
     if direction == "up":
-        y -= 1
+        y -= 2  # Error: should be 1
```

Comment:
The decrement for moving "up" is inconsistent with the other directions. It should likely be 1 for consistency unless there's a specific reason for it to be 2.
```diff
-        y -= 2  # Error: should be 1
+        y -= 1
```